fava/templates/_tree_table.html

```diff
@@ -31,8 +31,8 @@
     </p>
   </li>
   {% for account in ([account_node] if account_node.name else account_node.children) if account|should_show recursive %}
-  {% set balance = account.balance|cost_or_value %}
-  {% set balance_children = account.balance_children|cost_or_value %}
+  {% set balance = account.balance|cost_or_value(ledger._filters['time'].end_date) %}
```

Comment:
`_filters` is a "private" attribute of the ledger class, so it shouldn't be used

Reply:
Ok cool. How would you like me to expose the end_date in the FavaLedger class? Should I just create a property called end_date or do you want it to be more complicated than that? I could see a use for making all filters accessible, but then we're essentially renaming _filters to filters.

I suppose we could create a read-only property that exposed a copy of the filters to prevent accidental modification.